docs/listfilepages/globalfilestagpages/globalfilesprexxx.md

```diff
@@ -0,0 +1,2236 @@
+
+`     PREATT:6    `
+
+This would apply only if the PC had minimum base attack bonus of 6.
+
+------------------------------------------------------------------------
+
+### PREBASESIZE
+
+Require the character's base size is a particular size category, or in a particular range of size categories.
+
+#### Status
+
+Changed: `     size    ` used to be specified as the size name in words, i.e. `     Small    ` . Now is is specified as the `     SIZENAME    ` , i.e. `     S    ` .
+
+#### Syntax
+
+`     PREBASESIZEoperator:size    `
+
+`     operator    ` and `     size    ` are as per the [`      PRESIZE     ` tag](#presize) .
+
+#### Examples
+
+As per the [`      PRESIZE     ` tag](#presize)
+
+------------------------------------------------------------------------
+
+**<span id="prebirthplace"></span> Tag Name:** PREBIRTHPLACE:x
+
+**Variables Used (x):** Text (A birthplace name).
+
+**What it does:**
+
+Character's birthplace must match the listed text.
+
+**Example:**
+
+`     PREBIRTHPLACE:Klamath    `
+
+Character must have been born in "Klamath".
+
+------------------------------------------------------------------------
+
+<span id="precampaign"></span> \*\*\* New 5.15.2
+
+**Tag Name:** PRECAMPAIGN:x,y,y
+
+**Variables Used (x):** Number (Number of items required to pass)
+
+**Variables Used (y):** Text (Data set name as set by the CAMPAIGN tag).
+
+**Variables Used (y):** BOOKTYPE=Text (Matches data set BOOKTYPE tag).
+
+**Variables Used (y):** INCLUDES=Text (Data set name as set by the CAMPAIGN tag that is included in any selected campaign or one of its included sources). \*\*\* New 6.1.2
+
+**Variables Used (y):** INCLUDESBOOKTYPE=Text (Data set BOOKTYPE of any selected campaign or one of its included sources). \*\*\* New 6.1.2
+
+**What it does:**
+
+-   Sets prerequisites for LST objects based upon the data sets loaded into PCGen. If the set is loaded it passes, if not it doesn't.
+-   See PCC [PRECAMPAIGN](../datafilestagpages/datafilespcc.html#precampaign) docs for full information on this tag.
+
+------------------------------------------------------------------------
+
+<span id="precharactertype"></span> \*\*\* New 5.17.12
+
+**Tag Name:** PRECHARACTERTYPE:x,y,y
+
+**Variables Used (x):** Number (Number of character types required)
+
+**Variables Used (y):** Text (Character type.)
+
+**What it does:**
+
+Sets requisites for character type.
+
+**Examples:**
+
+PRECHARACTERTYPE:1,PC
+
+Character must be designated as a **PC** .
+
+PRECHARACTERTYPE:1,NPC,Monster
+
+Character must be a designated either as an **NPC** or as a **Monster** .
+
+------------------------------------------------------------------------
+
+**<span id="precheck"></span> Tag Name:** PRECHECK:x,y=z
+
+**Variables Used (x):** Number (The number of checks that must be equal to or greater than the numbers specified for the check to succeed).
+
+**Variables Used (y):** Text (A defined check from statsandchecks.lst - e.g. Fortitude or Willpower).
```

Comment:
Corrected in my working copy. Note inserted that `Fortitude`, `Reflex`, and `Will` are the names of the three usual checks.